### PR TITLE
Calculation

### PR DESCRIPTION
**Pull request by elenka-sh**
Bogdan, Task2Calculator If-Else performed, please check.

### CODE REVIEW

**Review by Hmeliumm — CHANGES_REQUESTED** [review]
Please remove Calculator in Calculator/src.
Move Calculator from Java/Calculator/Calculator/src/ to  Calculator/src.

File: Java/Calculator/Calculator/src/Calculator.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        System.out.println("Calculator");
+        System.out.println("Please enter First number into the console");
+        checkNumberOrString(sc);
+        Double number1 = sc.nextDouble();
+        System.out.println("Please input one the operand as '+ ,- ,* ,/ , % '");
+        char OperationSymbol = sc.next().charAt(0);
+        System.out.println("Please enter Second number into the console");
+        checkNumberOrString(sc);
+        Double number2 = sc.nextDouble();
+        final String SUM = "+";
```
Comment (Hmeliumm) [inline Java/Calculator/Calculator/src/Calculator.java:17]: constants should be at the beginning of file

File: Java/Calculator/Calculator/src/Calculator.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            System.out.println("User inputs unsupported integer. Repeat please inputting one the operand as '+ ,- ,* ,/ , % ");
+    }
+
+    private static void checkNumberOrString(Scanner sc) {
+        if (sc.hasNextDouble()) {
+            System.out.println();
+        } else {
+            System.out.println("WRONG!User inputs unsupported integer.");
+        }
+        System.out.println(sc.hasNextDouble() ? "" : "WRONG!User inputs unsupported integer.");
```
Comment (Hmeliumm) [inline Java/Calculator/Calculator/src/Calculator.java:42]: Please choose one variant (if..else or ternary operator)

**Review by Hmeliumm — CHANGES_REQUESTED** [review]

File: Java/Calculator/Calculator/src/Calculator.java (hunk 1 above)
Comment (elenka-sh) [reply Java/Calculator/Calculator/src/Calculator.java:17]: Done

File: Java/Calculator/Calculator/src/Calculator.java (hunk 2 above)
Comment (elenka-sh) [reply Java/Calculator/Calculator/src/Calculator.java:42]: Chenges done

**Pull request closed without merging**